src/pyocr/tesseract.py

```diff
@@ -25,6 +25,7 @@
 
 from . import builders
 from . import util
+from pyocr.builders import DigitBuilder
```

Comment:
This is to make sure that `DigitBuilder` is in `tesseract.__all__` for backward compatibility. Should I instead put a line `DigitBuilder = builders.DigitBuilder` ?